pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -702,6 +702,12 @@ func (w *Webhook) injectAutoInstruConfig(pod *corev1.Pod, config extractedPodLib
 		return err
 	}
 
+	if err := injector.mutatePod(pod); err != nil {
+		metrics.LibInjectionErrors.Inc("all", strconv.FormatBool(autoDetected), injectionType)
```

Comment:
can we define a tag value `any` instead of `all`?

Reply:
We already use `all` value [here](https://github.com/DataDog/datadog-agent/blob/de829214edf22fed981db33c7631448710faca8b/pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go#L726), I thought it'd be better to avoid adding new values, not sure if have `all` and `any` can cause confusion. 

WDYT?